src/main/resources/application.properties

```diff
@@ -1 +1,4 @@
 spring.application.name=GitHubAiAPI
+
+
```

Comment:
It's a good practice to follow a consistent naming convention for properties. The property 'Spring.demo.value' should be in lowercase.